.github/workflows/healthchecks_eventstore_ci.yml

```
@@ -69,7 +69,7 @@ jobs:
           --
           DataCollectionRunSettings.DataCollectors.DataCollector.Configuration.Format=opencover
       - name: Upload Coverage
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v5
```

This particular CI leg is red, lets see if it's affected by this change

```suggestion
        uses: codecov/codecov-action@v3
```